docs/code/reference.md

```diff
@@ -177,7 +178,7 @@ Opt for the second one, it's much cleaner.
 
 Gtk widgets are intended to respond to click events that can be described
 as "press + release" instead of "press". Use `toggle` and `release` events
-instead of `press`.
+instead of `press`. The "press + release" model allows for more granular actions.
```

Comment:
I don't think this is relevant for this PR?

Reply:
same as above